### DISCUSSION TITLE
columnChart example

### DISCUSSION THREAD

**Discussion by georgygo** [patrykandpatrick/vico#520] [Questions] [answered]
Hello,

I would like to create a chart similar to the one from the main page: 
![image](https://github.com/patrykandpatrick/vico/assets/126500718/b1a314a2-492d-4fde-843f-39e23962e994)

I was not able to find the code of the chart to take i as  an example. May you please let me know where to find this example's code?

If it's not available:
1) may you please advise how to create a bar chart with different entities grouped in the same bar as per example?
2) Link each line chart to the start Axis and bar chart to the left axis?

Thank you in advance?

**Comment by Gowsky** [comment, marked as answer]
Hi @georgygo, it's `Chart8` in the [sample module](https://github.com/patrykandpatrick/vico/blob/808daec29c31a3b940d08a0235bc4435e46c89d1/sample/src/main/java/com/patrykandpatrick/vico/sample/showcase/charts/Chart8.kt).

**Reply by georgygo** [reply]
Thanks for the reply! Now I could get the Chart in my project!
